submissions/BlueLamantine/html-css-popup/index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Hooli-style Popup</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="shortcut icon" href="img/favicon.png" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <nav class="topnav">
+      <ul class="topnav-list">
+        <li class="list-item">
+          <a href="#" class="nav-item">Mail</a>
+        </li>
+        <li class="list-item">
+          <a href="#" class="nav-item">Images</a>
+        </li>
+        <li class="list-item">
+          <input id="menu-toggle" type="checkbox" />
+          <label class="popup-btn nav-item" for="menu-toggle">
```

Comment:
If you write code using BEM methodology in the current line you have a BEM element `popup-btn` without parent BEM block `popup`, If not, don't mind about this comment.

Reply:
BEM methodology is welcomed, so I renamed alot of classes with BEM.